js/index.js

```diff
@@ -0,0 +1,26 @@
+import { getStartImagePath, getCoverPageInfo } from './util.js';
+
+var test_id = null
+var lan = null
+
+// JavaScript to make the first item clickable
+const firstItem = document.querySelector('.clickable-item:first-child a');
+
+if (firstItem) {
+    firstItem.addEventListener('click', function (e) {
+        e.preventDefault(); // Prevent the default link behavior
+        window.location.href = this.getAttribute('href'); // Navigate to start.html
+        test_id = 1;
+        lan = 'kor';
+    });
+}
```

Comment:
what happens if someone click second item instead of 1st item

Reply:
Nothing. I needed this page for testing. I plan to make it better later on.